### PR TITLE
documentation for the page param

### PR DESCRIPTION
**Pull request by edewit**
Adds missing documentation on how to use the page parameter

### CODE REVIEW

**Comment by edewit** [comment]
@lfryc care to have a look if this makes sense?

File: docs/guides/aerogear-cordova/AerogearCordovaPush.asciidoc [hunk 1]
```diff
@@ -206,6 +206,20 @@ function onNotification(event) {
...
+[source,javascript]
+----
+"message": {
+    "windows": {
+      "page": "cordova"
+    }
+}
+----
+
+The sender will use this to trigger a 'special internal' page within the plugin that will call your javascript callback and parse the user-data. Without the app will come to foreground without invoking the javascript callback like you are used on the other platforms.
```
Comment (lfryc) [inline docs/guides/aerogear-cordova/AerogearCordovaPush.asciidoc]: `Without the app will come to foreground without invoking the javascript callback`

this is probably not correctly phrased..

**Comment by lfryc** [comment]
:+1: Seems fine to me. Merging...

**Comment by lfryc** [comment]
Landed: https://github.com/aerogear/aerogear.org/commit/307f9cd873241cf5effd2a82aaf6f684a74af023 :rocket: :moon:

**Pull request closed without merging**